src/App.js

```diff
@@ -23,19 +27,28 @@ function App() {
     });
   }, []);
 
+  useEffect(() => {
```

Comment:
can add comments here explaining what this useEffect does

Reply:
Comments added.